### PR TITLE
Add in SV calling Tasks [PBSV, Sniffles]

### PR DESCRIPTION
**Pull request by SHuang-Broad**
After #3 .

### CODE REVIEW

File: data/LRWholeGenomeSingleSample/TestWithChrM.json [hunk 1]
```diff
@@ -10,5 +10,8 @@
     "LRWholeGenomeSingleSample.ref_fasta":     "gs://broad-dsde-methods-long-reads/resources/references/grch38/Homo_sapiens_assembly38.fasta",
     "LRWholeGenomeSingleSample.ref_fasta_fai": "gs://broad-dsde-methods-long-reads/resources/references/grch38/Homo_sapiens_assembly38.fasta.fai",
     "LRWholeGenomeSingleSample.ref_dict":      "gs://broad-dsde-methods-long-reads/resources/references/grch38/Homo_sapiens_assembly38.dict",
+
+    "LRWholeGenomeSingleSample.tandem_repeat_bed": "gs://broad-dsde-methods-long-reads/2019-Aug-PacBio-Nature-Biotech-Paper/human_GRCh38_no_alt_analysis_set.trf.bed",
```
Comment (kvg) [inline data/LRWholeGenomeSingleSample/TestWithChrM.json]: We should copy the tandem repeat finder file over to gs://broad-dsde-methods-long-reads/resources/references/grch38/ .

File: wdl/CallSV.wdl [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    }
+
+    Int disk_size = ceil(size(bam, "GB")) + 10
+
+    String out_prefix = basename(bam)
+
+    command <<<
+        set -euxo pipefail
+
+        num_core=$(cat /proc/cpuinfo | awk '/^processor/{print $3}' | wc -l)
```
Comment (kvg) [inline wdl/CallSV.wdl]: Ahh, this is interesting.  I'm curious - if we're setting the number of cores below in the runtime section, why do we need to detect the number of cores in the command itself?  Is this so we can change the runtime section without changing the code for the command section, and thus call caching will continue to work even if we change the number of CPUs between invocations?

File: data/LRWholeGenomeSingleSample/TestWithChrM.json (hunk 1 above)
Comment (SHuang-Broad) [reply data/LRWholeGenomeSingleSample/TestWithChrM.json]: I've thought about that, but I'm not sure how we should provide information on how each file was produced.

A real, related situation that I face: while test running this subworkflow before issuing this PR, is that the reference `hs37d5.fasta` does not work well with `gs://broad-dsde-methods-long-reads/2019-Aug-PacBio-Nature-Biotech-Paper/human_hs37d5.trf.bed` for PBSV. The reason is that `hs37d5.fasta` contains the decoy contig at the end (hence 86 contigs), while the SVSIG file produced (by `pbsv discover`) with that TRF annotation requests the reference file  in the following `pbsv call` stage to have exactly 85 contigs in the reference. Annoying, but reality...

The point is that we should have a way of keeping track of how each reference related annotation is produced (if downloaded, note source; if custom, note program, parameters and inputs used).
What do you think?

File: wdl/CallSV.wdl (hunk 2 above)
Comment (SHuang-Broad) [reply wdl/CallSV.wdl]: Exactly.
The default runtime (or VM specification) can be overriden by users.
But the actual command in the workflow will be OK with how many ever (but still reasonable) CPUs available to the container.

One trivial issue though, is that the PBSV docker image on quay.io is built with Busybox (embeded Linux, very lean), so other mechanisms for deciding CPU numbers are unavailable....

**Pull request merged**